Source/Orts.Simulation/Simulation/RollingStocks/MSTSWagon.cs

```diff
@@ -651,330 +641,118 @@ public virtual void LoadFromWagFile(string wagFilePath)
-                    }
-                    else
-                    {
-                        LoadFullMaxHandbrakeForceN = MaxHandbrakeForceN;
-                    }
-
-                    if (FreightAnimations.FullPhysicsStaticOne.FullStaticCentreOfGravityM_Y > 0)
-                    {
-                        LoadFullCentreOfGravityM_Y = FreightAnimations.FullPhysicsStaticOne.FullStaticCentreOfGravityM_Y;
-                    }
-                    else
-                    {
-                        LoadFullCentreOfGravityM_Y = CentreOfGravityM.Y;
-                    }
-
-                    if (FreightAnimations.FullPhysicsStaticOne.FullStaticRelayValveRatio > 0)
-                    {
-                        LoadFullRelayValveRatio = FreightAnimations.FullPhysicsStaticOne.FullStaticRelayValveRatio;
-                    }
-                    else if (BrakeSystem is AirSinglePipe brakes)
-                    {
-                        LoadFullRelayValveRatio = brakes.RelayValveRatio;
-                    }
-
-                    if (FreightAnimations.FullPhysicsStaticOne.FullStaticInshotPSI > 0)
-                    {
-                        LoadFullInshotPSI = FreightAnimations.FullPhysicsStaticOne.FullStaticInshotPSI;
-                    }
-                    else if (BrakeSystem is AirSinglePipe brakes)
-                    {
-                        LoadFullInshotPSI = brakes.RelayValveInshotPSI;
-                    }
-                }
+                setIfPositive(ref LoadFullORTSDavis_A, FreightAnimations.FullPhysicsStaticOne?.FullStaticORTSDavis_A);
+                setIfPositive(ref LoadFullORTSDavis_B, FreightAnimations.FullPhysicsStaticOne?.FullStaticORTSDavis_B);
+                setIfPositive(ref LoadFullORTSDavis_C, FreightAnimations.FullPhysicsStaticOne?.FullStaticORTSDavis_C);
+                setIfPositive(ref LoadFullDavisDragConstant, FreightAnimations.FullPhysicsStaticOne?.FullStaticORTSDavisDragConstant);
+                setIfPositive(ref LoadFullWagonFrontalAreaM2, FreightAnimations.FullPhysicsStaticOne?.FullStaticORTSWagonFrontalAreaM2);
+                setIfPositive(ref LoadFullMaxBrakeForceN, FreightAnimations.FullPhysicsStaticOne?.FullStaticMaxBrakeShoeForceN);
+                setIfPositive(ref LoadFullMaxBrakeForceN, FreightAnimations.FullPhysicsStaticOne?.FullStaticMaxBrakeForceN);
+                setIfPositive(ref LoadFullMaxHandbrakeForceN, FreightAnimations.FullPhysicsStaticOne?.FullStaticMaxHandbrakeForceN);
+                setIfPositive(ref LoadFullCentreOfGravityM_Y, FreightAnimations.FullPhysicsStaticOne?.FullStaticCentreOfGravityM_Y);
+                setIfPositive(ref LoadFullRelayValveRatio, FreightAnimations.FullPhysicsStaticOne?.FullStaticRelayValveRatio);
+                setIfPositive(ref LoadFullInshotPSI, FreightAnimations.FullPhysicsStaticOne?.FullStaticInshotPSI);
```

Comment:
This one should be `setIfNonZero` as well fyi. (The other places where inshot is set are already fine.)
```suggestion
                setIfNonZero(ref LoadFullInshotPSI, FreightAnimations.FullPhysicsStaticOne?.FullStaticInshotPSI);
```